scripts/support/create_ge_images/test_only.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+#
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FI-Core project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+
+cd $(dirname $0)
+. create_template.sh
+# Parameters: name
+check_params $*
+
+DIR=$IMAGES_DIR/$1
+user=$(cat $DIR/dist_type)
+# Delete old VM if it exists
+nova delete $(cat $DIR/last_vm 2>/dev/null) >/dev/null 2>&1  || true
+rm -f $DIR/last_vm
+test_template $1 $user
```

Comment:
IMAGE_DIR is defined at loading . create_template.sh. It is the directory of the user where the files are (~/create_ge_images)
user is passed to the test script. Is the username of the account where it can be connected. This account is dependend of the distribution, but for the five base images it coincides with the name of the distribution (ubuntu, debian, centos)